demo

```diff
@@ -1,13 +1,13 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Why move to `bash`, from what I understand `sh` is more widely available across operating systems

Reply:
This was interesting. You get the following error when you do not use `bash` because `sh` does not support that `if` expression syntax.
https://stackoverflow.com/questions/3411048/unexpected-operator-in-shell-programming

```
╰─$ cat ./demo
#!/bin/sh

if [ "$1" == "start" ]; then
  echo "Downloading pre-built docker images"
  docker-compose -f examples/rails-app/demo.yml pull
  echo "Setting up and deploying Super Graph and the demo Rails app"
  docker-compose -f examples/rails-app/demo.yml run rails_app rake db:create db:migrate db:seed
  docker-compose -f examples/rails-app/demo.yml up
elif [ "$1" == "stop" ]; then
  docker-compose -f examples/rails-app/demo.yml down
else
  echo "./demo [start|stop]"
fi

$ ./demo start
./demo: 3: [: start: unexpected operator
./demo: 9: [: start: unexpected operator
./demo [start|stop]
```